formats/AtomFormat.php

```diff
@@ -1,21 +1,30 @@
 <?php
 /**
-* Atom
-* Documentation Source http://en.wikipedia.org/wiki/Atom_%28standard%29 and
-* http://tools.ietf.org/html/rfc4287
-*/
+ * AtomFormat - RFC 4287: The Atom Syndication Format
+ * https://tools.ietf.org/html/rfc4287
+ *
+ * Validator:
+ * https://validator.w3.org/feed/
+ */
 class AtomFormat extends FormatAbstract{
+	const LIMIT_TITLE = 140;
+
 	public function stringify(){
-		$https = isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on' ? 's' : '';
-		$httpHost = isset($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : '';
-		$httpInfo = isset($_SERVER['PATH_INFO']) ? $_SERVER['PATH_INFO'] : '';
+		$urlScheme	= (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') ? 'https://' : 'http://';
```

Comment:
This is not a scheme. A scheme mustn't contain `://`

```suggestion
		$urlScheme = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on') ? 'https' : 'http';
```

Reply:
That's the first time I hear about that. I've seen a lot technical documentation that refer to schemes having the "://" part - sometimes just ":" appended depending on the subject. This makes it clear whether it's about a location or something else.

Anyways, this is about the whole prefix part including "://". It's easier to use the actual prefix than to just store an "s" and append it somewhere later where its purpose is not clear. Think about maintenance.

Reply:
[RFC 3986](https://tools.ietf.org/html/rfc3986) defines the components of URIs, see [page 16](https://tools.ietf.org/html/rfc3986#page-16)

>         foo://example.com:8042/over/there?name=ferret#nose
>         \_/   \______________/\_________/ \_________/ \__/
>          |           |            |            |        |
>       scheme     authority       path        query   fragment
>          |   _____________________|__
>         / \ /                        \
>         urn:example:animal:ferret:nose

`parse_url` uses the same notation: http://php.net/manual/en/function.parse-url.php

>Anyways, this is about the whole prefix part including "://". It's easier to use the actual prefix than to just store an "s" and append it somewhere later where its purpose is not clear. Think about maintenance.

Don't get me wrong, I find your solution way better than the previous one. It's just that the scheme has a specific meaning which doesn't include `://`.

Reply:
No sweat, just renamed it to avoid confusions about this.

Reply:
Thanks!